rasa/utils/tensorflow/models.py

```diff
@@ -268,6 +286,34 @@ def rasa_predict(
         )
         return outputs
 
+    def run_inference(
+        self, model_data: RasaModelData, batch_size: Union[int, List[int]] = 1
+    ) -> Dict[Text, Union[np.ndarray, Dict[Text, Any]]]:
+        """Implements bulk inferencing through the model.
+
+        Args:
+            model_data: Input data to be fed to the model.
+            batch_size: Size of batches that the generator should create.
+
+        Returns:
+            Model outputs corresponding to the inputs fed.
+        """
+        outputs = {}
+        (data_generator, _,) = rasa.utils.train_utils.create_data_generators(
+            model_data=model_data, batch_sizes=batch_size, epochs=1, shuffle=False,
+        )
+        data_iterator = iter(data_generator)
+        while True:
+            try:
+                # Only want x, since y is always None out of our data generators
```

Comment:
Could you make this comment a bit clearer? The thing I didn't understand when I read it was the x vs y thing. A generator always gives an (x, y) pair, where x are inputs and y are labels?